src/emulator/env.ts

```diff
@@ -46,6 +46,9 @@ export function setEnvVarsForEmulators(
         break;
       case Emulators.DATACONNECT:
         env[Constants.FIREBASE_DATACONNECT_EMULATOR_HOST] = host;
```

Comment:
I *think* you need to include the protocol for the emulator, so:
```suggestion
        env[Constants.FIREBASE_DATACONNECT_EMULATOR_HOST] = `http://${host}`;
```

Reply:
For the record the protocol should never be set, see https://github.com/firebase/firebase-tools/pull/8396